app/models/admin.rb

```diff
@@ -1,6 +1,6 @@
 class Admin < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
-  devise :database_authenticatable,
+  devise :database_authenticatable, :registerable,
```

Comment:
"Registerable" there means that any user can come and register himself as an admin. It's not an option.